### PR TITLE
Added ssh-jump plugin

### PR DESCRIPTION
**Pull request by yokawasa**
Adds [ssh-jump](https://github.com/yokawasa/kubectl-plugin-ssh-jump) plugin

-----

**Checklist for plugin developers:**

- [x] Read the [Plugin Naming Guide](https://github.com/GoogleContainerTools/krew/tree/master/docs/NAMING_GUIDE.md) (for new plugins)
- [x] Verify the installation from URL or a local archive works (`kubectl krew install --manifest=[...] --archive=[...]`)

### CODE REVIEW

**Comment by yokawasa** [comment]
I just signed CLA! Please review

File: plugins/ssh-jump.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ssh-jump         # plugin name must match your manifest file name (e.g. foo.yaml)
```
Comment (ahmetb) [inline plugins/ssh-jump.yaml]: do you mind deleting these comments leftover from the template?

File: plugins/ssh-jump.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  # specify installation script for linux and darwin (macOS)
+  - selector:             # a regular Kubernetes selector
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    # url for downloading the package archive:
+    uri: https://github.com/yokawasa/kubectl-plugin-ssh-jump/archive/0.1.0.zip
+    # sha256sum of the file downloaded above:
+    sha256: "302a6df87fdbb75c14a275b092209d4191abe5b4cc802a77eb963f127c8edc43"
+    files:                     # copy the used files out of the zip archive
+    - from: "/kubectl-plugin-ssh-jump*/kubectl-ssh-jump" # path to the files extracted from archive
```
Comment (ahmetb) [inline plugins/ssh-jump.yaml]: you can probably just do `./*/kubectl-ssh-jump` as there won't be other files in the zip file downloaded from github.

File: plugins/ssh-jump.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      to: "."               # '.' refers to the root of plugin install directory
+    bin: "./kubectl-ssh-jump"  # path to the plugin executable after copying files above
+  shortDescription: A kubectl plugin to SSH into Kubernetes nodes using a SSH jump host Pod
+  # (optional) use caveats field to show post-installation recommendations
+  caveats: |
+    This plugin needs the following programs:
+    * ssh(1)
+    * ssh-agent(1)
+  description: |
```
Comment (ahmetb) [inline plugins/ssh-jump.yaml]: Maybe also link to your repo for docs from here or from `caveats:`, so users read it after they install the plugin?

Comment (yokawasa) [reply plugins/ssh-jump.yaml]: I added the document link in `caveats` part. Please review

Comment (ahmetb) [reply plugins/ssh-jump.yaml]: Can you please restore the requirements (required tools) in the `caveats` section? It was very useful.

Comment (yokawasa) [reply plugins/ssh-jump.yaml]: Thank you for the review! I restore the requirements

**Pull request merged**